### DISCUSSION TITLE
Missing emissions from woody biomass. Electricitymap does not show real but only accounted emissions.

### DISCUSSION THREAD

**Discussion by ofenaus** [electricitymaps/electricitymaps-contrib#3721] [General]
According to the quoted source document for emissions ("IPCC 2014") the direct emissions for biomass (dedicated) are not 0. Instead the footnote says they are "significant" - which is correct, actually they are higher than the emissions of coal. If they were 0, the IPCC would have used this value like they did for hydropower or geothermal. But they are not.

Currently electricitymap is using "accounted emissions", not real emissions:

https://www.chathamhouse.org/2017/02/woody-biomass-power-and-heat/2-accounting-biomass-carbon-emissions

I strongly recommend to have a look into the most recent scientific papers on this topic, e.g.

https://www.researchgate.net/publication/357375767_Primeval_natural_and_commercial_forests_in_the_context_of_biodiversity_and_climate_protection_Part_2_The_narrative_of_the_climate_neutrality_of_wood_as_a_resource

Especially the blue box on page 33 outlines the issue of non-existant millions tons of carbon dioxide from woody biomass.

The IPCC itself says that direct(!) emissions from wood are actually amongst the highest amongst all energy sources (page 2.23).

https://www.ipcc-nggip.iges.or.jp/public/2006gl/pdf/2_Volume2/V2_2_Ch2_Stationary_Combustion.pdf

More sources:
https://iopscience.iop.org/article/10.1088/1748-9326/aaa512/pdf
https://publications.jrc.ec.europa.eu/repository/handle/JRC122719
https://www.wwf.eu/?2128466%2F500-scientists-tell-EU-to-end-tree-burning-for-energy

The IPCC models you are using did NOT assess use of forest-derived wood pellets for electricity:

https://www.nrdc.org/experts/sasha-stashwick/what-ipcc-really-says-forest-biomass-climate-change

This becomes VERY clear if you have a look at the FAQ published by the NGGIP:

https://www.ipcc-nggip.iges.or.jp/faq/faq.html

Q2-10:

The IPCC guidelines provide the simplifying assumption (Tier 1 method) regarding CO2 emissions and removals associated with annual biomass (e.g., corn, colza) that over the course of a year, the CO2 emissions from the combustion/oxidation/decay of annual biomass are balanced by carbon uptake prior to harvest, within the uncertainties of the estimates, so the net emission is zero.

This assumption is ONLY valid for annual biomass. Woody biomass has never been declared net-zero by the IPCC. Net-zero biomass is only an accounting rule, not a scientific fact:

https://ens.dk/sites/ens.dk/files/Bioenergi/biomasseanalyse_final_ren_eng.pdf

At the moment electricitymap is not providing accurate data and is supporting greenwashing of dirty biomass. Your work is actually contradicting the efforts of pretty much every environmental protection organization:

Fridays For Future:
https://twitter.com/BeyondBurning/status/1470364778631684108
Robin Wood:
https://www.robinwood.de/pressemitteilungen/holz-statt-kohle-klimaschwindel-bei-der-energiewende-nicht-f%c3%b6rdern
WWF:
https://www.wwf.eu/?2128466%2F500-scientists-tell-EU-to-end-tree-burning-for-energy
EEB:
https://eeb.org/wp-content/uploads/2021/09/Where-theres-fire-theres-smoke_domestic-heating-study_2021.pdf
FERN:
https://www.fern.org/fileadmin/uploads/fern/Documents/briefingnote%20bioenergy_German.pdf
Greenpeace:
https://www.greenpeace.org/eu-unit/issues/nature-food/45768/5-reasons-why-burning-trees-for-energy-is-bad-for-climate-people-and-nature/
Somo:
https://www.somo.nl/wood-pellet-damage/
...

To further support this I would like to draw your attention to the ongoing debate on the taxonomy for gas and nuclear energy. If the taxonomy for gas would be ratified:

will you continue use the current emission values for gas or
will you present emissions according to the taxonomy?

And will you only apply this for countries inside the EU?

If you apply the logic you are currently using for woody biomass, countries using significant amounts of gas would instantly have to be reported "green" without any actual change to their energy production.

If this appears like greenwashing to you: that is exactly what is currently done for reporting of emissions from woody biomass.

Please take action and decide, whether you want to monitor "accounted emissions" or actual emissions. And please be transparent towards your users about your decision and the methodology used.

**Comment by pierresegonne** [comment]
Hey @ofenaus thank you for raising this point here.
The emap team generally looks at issues rather than at discussions to interact with the community. The discussions are supposed to give contributors a place to exchange that is less restricted than issues. That explains why we've not been reactive to your initial point. For answers, please refer to the issues you've opened.

**Comment by ofenaus** [comment]
Addendum:

As already mentioned, millions of tons of carbon dioxide from woody biomass (e.g. in Drax power station in UK) is not shown on electricity map as electricity map does NOT use effective emissions but accounting rules, making electricity map effectively useless.

On 17th May 2022 the European Parliament’s Environment Committee recommended the following:

    A definition for primary woody biomass (PWB), meaning biomass sourced directly from forests, was added for the first time to the RED. The definition however includes exemptions for forests affected by fires, pests and disease, carving out significant potential loopholes for the use of PWB.

    PWB will no longer qualify as counting towards Member States’ renewable energy targets, with a number of exemptions: bioenergy produced at installations smaller than 7.5 MW heat input, installations with BECCS technology, and electricity-only installations less than 20 MW in the EU’s

    PWB will no longer receive subsidies under the RED, again with a set of exemptions.

    The cascading principle for biomass now needs to be taken into account, including for secondary woody biomass: wood should first be used for longer lived products, and only if there are no options for this use should it be burned for energy.

https://forestdefenders.eu/european-parliaments-environment-committee-recommends-curtailing-burning-forest-biomass-for-renewable-energy/
https://www.nytimes.com/2022/05/17/climate/eu-burning-wood-electricity.html?smid=tw-share

If this recommendation finds its way into the new RED-directive, electricity map will finally have to refrain from using political accounting rules for emissions from biomass and instead be forced to show real emissions - which will dramatically change the current numbers, especially for countries with high use of primary woody biomass for energy generation.

**Comment by madsnedergaard** [comment]
Locking this in favor of the issue #3727 :)
